### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by chrisisla**
First draft of a template that we can use for CDM Pull Requests

### CODE REVIEW

**Comment by chrisisla** [comment]
Hi @dshoneisda and @lolabeis ,
I've created this draft pull request template based upon the feedback we have had to date. Please take a look and let me know what you think. Too many hints? Too few hints? Too wordy? Not wordy enough? Are we missing any sections/detail that should be added to the page?
Thanks!

**Review by dshoneisda — APPROVED** [review]
looks ok to me- thanks
